src/resources/vue/pages/index.vue

```diff
@@ -3,9 +3,9 @@
 </template>
 
 <script>
-
+export default {
+  name: "Index",
+};
 </script>
 
-<style>
-
-</style>
+<style></style>
```

Comment:
Geen lege tags

Reply:
Dit is letterlijk de standaard page, dit was al zo